02-getting-account-data/README.md

```diff
@@ -0,0 +1,10 @@
+# 01 - Creating a Transaction
+
+The goal of this second round is to try and create a code sample that illustrates a developer should instantiate an `Account` object with the new Web Client SDKs. 
+
+The developer will need to:
+
+- Setup an `Account` class to access an account's assets and resources.
+- Use a `Session` object to instantiate the `Account` object when the user is logged in.
```

Comment:
A session object should not be needed

Reply:
True! It won't be needed, but you will be able to instantiate an account from a session object.

Reply:
Maybe we focus this round (02-getting-account-data) to an account instantiated from the Account Kit?

I'm trying to limit the scope of each round down to the minimums, so we can focus on one aspect at a time. We can open a round for various aspects of the Session Kit and Contract Kit to address ideas for those.